### DISCUSSION TITLE
Latest Release

### DISCUSSION THREAD

**Discussion by Chibane690** [getredash/redash#6990] [Q&A]
I noticed on the Redash GitHub page that several test versions have been released, with the latest being 24.05.0-dev and the latest stable release being 10.1.0. When can we expect the latest test version to be available as a stable release, and what new features can we look forward to?

**Comment by zachliu** [comment]
i have the same question...

to this day i'm still using the long-dead [mozilla's fork](https://github.com/mozilla/redash) because they have implemented the [Schema Viewer Drawer](https://github.com/mozilla/redash/commit/9695b855ceb2aa6ea442c5a0b31f9bb2c1b436c8) feature which allows users to see column metatdata such as data types
